docker-compose.yml

```diff
@@ -5,15 +5,18 @@ services:
     command: --default-authentication-plugin=mysql_native_password
     restart: always 
     environment: 
```

Comment:
have you checked https://docs.docker.com/compose/environment-variables/#the-env-file